doc/api/vm.md

```diff
@@ -429,6 +429,30 @@ local scope, so the value `localVar` is changed. In this way
 `vm.runInThisContext()` is much like an [indirect `eval()` call][], e.g.
 `(0,eval)('code')`.
 
+## vm.createModule(code[, options])
+<!-- YAML
+added:
```

Comment:
we generally use `REPLACEME` here because that will be picked up by the release tooling